README.md

```diff
@@ -1,10 +1,25 @@
-# rustls-openssl-compat
+# Rustls OpenSSL Compatibility Layer
 
-This is the planned home of several rustls to OpenSSL compatibility layers.
-Currently here:
+OpenSSL consists of several libraries. We offer a safer version of OpenSSL's `libssl` (TLS support), and we intend to eventually offer a safer version of OpenSSL's `libcrypto` (cryptography support).
```

Comment:
> and we intend to eventually offer a safer version of OpenSSL's `libcrypto` (cryptography support).

I'm not sure that is super feasible, given how huge and crazy the API there is. I think maybe reword this to say about _using_ libcrypto (which is, AIUI, desirable for various reasons like taking advantage of openssl's fips cert on a wider variety of platforms than aws-lc's covers).